bycycle/utils.py

```diff
@@ -0,0 +1,90 @@
+"""Plotting utility functions."""
+
+import numpy as np
+
+###################################################################################################
+###################################################################################################
+
+
+def limit_df(df, fs, xlim):
```

Comment:
`xlim`, as a name, is really related to plotting, and these functions are now organized as general utilities.
Also, the current API makes it harder to apply either just a start time, or just a stop time, which I think could potentially be useful? 

What do we think about this possible API:
`def limit_df(df, fs, start, stop):`

If so, and whatever we choose, I think it should be matched in `limit_sig_times`, and if it keeps the 2 item list, maybe update the name. Maybe `limits`?

Reply:
I can break xlim into start/stop args. Do you think defaulting start to 0 and stop to -1 makes sense to allow only one variable to be defined? I'll need to change the code to work with -1.

Reply:
My first thought was `(start=None, stop=None)`, and then they get applied individually, as like `if start: # limit start`, and `if stop: # limit stop`. I'm not sure start / stop as 0 / -1 would work very cleanly without more work, because the current values at time values, in seconds (right?), where a 0/-1 look like indices. Supporting both or defaulting to use indices seems like mixed convention / possibly confusing.